docs/api/apiv3/paths/project_favorite.yml

```diff
@@ -0,0 +1,78 @@
+# /api/v3/projects/{id}/favorite
+---
+delete:
+  parameters:
+  - description: Project id
+    example: '1'
+    in: path
+    name: id
+    required: true
+    schema:
+      type: integer
+  responses:
+    '204':
+      description: |-
+        Returned if the project was successfully removed from favorites.
+      headers: {}
+    '404':
+      content:
+        application/hal+json:
+          schema:
+            $ref: "../components/schemas/error_response.yml"
+          examples:
+            response:
+              value:
+                _type: Error
+                errorIdentifier: urn:openproject-org:api:v3:errors:NotFound
+                message: The requested resource could not be found.
+      description: |-
+        Returned if the project does not exist or the client does not have sufficient permissions
+        to see it.
+
+        **Required permission:** view project
```

Comment:
View project is a public permission, so even anonymous gets it. But they should not be able to mark a project as favorite.